client/app/components/ParameterValueInput.less

```diff
@@ -1,50 +1,80 @@
 @import '~antd/lib/input-number/style/index'; // for ant @vars
 
+@input-dirty: #fffce1;
+
 .parameter-input {
   display: inline-block;
   position: relative;
-  width: 195px;
 
-  &[data-dirty] {
-    .@{ant-prefix}-input {
-      padding-right: 58px !important;
-    }
+  .@{ant-prefix}-input {
+    width: 195px;
```

Comment:
This is also being applied to DateRanges :sweat_smile: 

![date-time-range](https://user-images.githubusercontent.com/3356951/60890366-8598ed80-a231-11e9-869a-8504d4a05f77.png)

Reply:
Whoops! https://github.com/getredash/redash/pull/3907/commits/bb197311ea2c1d1091225e90b0949e405fc183f2